students/JaeKim/lesson03/assignment/test_basic_operations.py

```diff
@@ -0,0 +1,185 @@
+import customers_model as cm
+import pytest
+from customers_model import TestCustomer
+from basic_operations import add_customer, search_customer, delete_customer, \
+                             update_customer_credit, list_active_customers
+
+"""
+Module docstring
+"""
+
+OK_CUSTOMER = {'customer_id': '12345',
+               'name': 'Suzie',
+               'lastname': 'Edgar',
+               'home_address': '123 Browncroft Blvd, Rochester, NY 97235',
+               'phone_number': '234-123-4567',
+               'email_address': 'suzie@hotmail.com',
+               'status': True,
+               'credit_limit': 40,
+               'stage': 'dev'}
+
+
+def setup_test_db():
+    """
+    Drops test_customer and re-creates an empty test_customer db
+    """
+    # Cleans up Test table from previously failed tests
+    teardown_test_db()
+
+    # Create a Test table
+    cm.database.create_tables([
+        cm.TestCustomer
+    ])
+    cm.database.close()
+
+
+def teardown_test_db():
+    """
+    Drops test_customer, used for teardown
+    """
+    cm.database.drop_tables([
+        cm.TestCustomer
+    ])
+    cm.database.close()
+
+
+def test_add_ok_customer():
+    """
+    sets up a new db, adds customer, runs assertions and tears down db
+    """
+    setup_test_db()
+    add_customer(**OK_CUSTOMER)
+    valid_customer = TestCustomer.get(TestCustomer.customer_id == OK_CUSTOMER['customer_id'])
+
+    assert valid_customer.first_name == OK_CUSTOMER['name']
+    assert valid_customer.customer_id == OK_CUSTOMER['customer_id']
+    assert valid_customer.last_name == OK_CUSTOMER['lastname']
+    assert valid_customer.home_address == OK_CUSTOMER['home_address']
+    assert valid_customer.phone_number == OK_CUSTOMER['phone_number']
+    assert valid_customer.email_address == OK_CUSTOMER['email_address']
+    assert valid_customer.status == OK_CUSTOMER['status']
+    assert valid_customer.credit_limit == OK_CUSTOMER['credit_limit']
+
+    teardown_test_db()
```

Comment:
You don't have to do the teardown method here, since you do it in your setup method.